cnd/src/swap_protocols/facade.rs

```diff
@@ -123,6 +126,16 @@ impl Facade {
             .initiate_communication(id, peer, role, digest, identities)
             .await
     }
+
+    /// True if expiry time has elapsed according to the Bitcoin connector.
+    pub async fn can_refund_bitcoin(&self, expiry: Timestamp) -> bool {
```

Comment:
Nit: This function only calls through to `has_passed`, why is it named differently?

Reply:
The facade method has not been used yet; I was looking into the future and thinking that the caller of this method would like to see _why_ we are checking if the expiry has passed, hence this method name. The connector however just exposes lower level functionality to check if a time has passed, hence the trait name.